### PR TITLE
Updating mysqli: set_charset

### PR DESCRIPTION
**Pull request by kamil-tekiela**
I am not sure if the first note on this page is still relevant.

### CODE REVIEW

**Comment by zors1** [comment]
I know that it's not directly related to this PR, but isn't `&style.oop;` missing a hyphen?

image

**Comment by kamil-tekiela** [comment]
@zors1 Can you explain where do you think the hyphen is missing? What should it look like with a hyphen?

**Comment by zors1** [comment]
>  Do you mean that is should be "Object-oriented style"?

Yes, I'm sorry my previous comment wasn't clear enough.

**Comment by kamil-tekiela** [comment]
Then, yes. It should be "Object-oriented", but that's not part of this PR.

File: reference/mysqli/mysqli/set-charset.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
      <term><parameter>charset</parameter></term>
      <listitem>
       <para>
-       The charset to be set as default.
+       The character set to be set.
```
Comment (afilina) [inline reference/mysqli/mysqli/set-charset.xml:38]: Maybe a nitpick, but having the word "set" twice makes it a bit harder to read. Perhaps "The desired character set." or "The character set to be used". We already say "Sets the character set to be used" in the description; the second option merely reiterates that in context.

**Review by afilina — CHANGES_REQUESTED** [review]
Other than the one comment, LGTM.

File: reference/mysqli/mysqli/set-charset.xml (hunk 1 above)
Comment (kamil-tekiela) [reply reference/mysqli/mysqli/set-charset.xml]: I agree. I used "The desired character set."

**Review by afilina — APPROVED** [review]

**Pull request merged**